.github/workflows/build.yml

```diff
@@ -9,11 +9,22 @@ on:
 jobs:
   linux:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        encoding:
+          - ICONV
+          - ICU
+    env:
+      ENCODING_TYPE: ${{matrix.encoding}}
     steps:
       - uses: actions/checkout@v3
       - name: restore
         run: |
           sudo apt-get install -y libgtest-dev
```

Comment:
I see I didn't catch this before, but we should always run `sudo apt-get update` before installing any packages. See for example the [Installing software on Ubuntu runners](https://docs.github.com/en/enterprise-cloud@latest/actions/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners) section in the official GitHub Actions docs:

> **Note**: Always run `sudo apt-get update` before installing a package. In case the apt index is stale, this command fetches and re-indexes any available packages, which helps prevent package installation failures.

Reply:
Looks like fixed in e5e45dbb.